.travis.yml

```diff
@@ -5,9 +5,22 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "pypy"
 
 sudo: false
 install:
+  - |
+        if [ "$TRAVIS_PYTHON_VERSION" = "pypy" ]; then
+          export PYENV_ROOT="$HOME/.pyenv"
+          if [ -f "$PYENV_ROOT/bin/pyenv" ]; then
+            pushd "$PYENV_ROOT" && git pull && popd
+          else
+            rm -rf "$PYENV_ROOT" && git clone --depth 1 https://github.com/yyuu/pyenv.git "$PYENV_ROOT"
+          fi
+          export PYPY_VERSION="5.6.0"
+          "$PYENV_ROOT/bin/pyenv" install --skip-existing "pypy-$PYPY_VERSION"
+          virtualenv --python="$PYENV_ROOT/versions/pypy-$PYPY_VERSION/bin/python" "$HOME/virtualenvs/pypy-$PYPY_VERSION"
+        fi
```

Comment:
This change has not resolved the problem. You're still not `pip install`ing the dependences into the virtualenv.

Please, take a look at what certitude does and consider that approach.